core/kazoo_services/src/modules/kz_services_payment_tokens.erl

```diff
@@ -27,7 +27,7 @@ fetch(?NE_BINARY = AccountId) ->
     fetch(kz_services:fetch(AccountId));
 fetch(Services) ->
     ServicesJObj = kz_services:services_jobj(Services),
-    kzd_services:payment_token(ServicesJObj, []).
+    kzd_services:payment_tokens(ServicesJObj, []).
```

Comment:
Good catch! Also `payment_tokens` is a JObj not a list, so please changes it to:

```
kzd_services:payment_tokens(ServicesJObj, kz_json:new()).
```

Also change spec's return type to `kz_json:object()`.